pkg/services/sqlstore/alert_notification.go

```diff
@@ -179,8 +197,19 @@ func UpdateAlertNotification(cmd *m.UpdateAlertNotificationCommand) error {
 		current.Name = cmd.Name
 		current.Type = cmd.Type
 		current.IsDefault = cmd.IsDefault
+		current.NotifyOnce = cmd.NotifyOnce
 
-		sess.UseBool("is_default")
+		if cmd.Frequency == "" {
+			return fmt.Errorf("Alert notification frequency required")
+		}
+
+		frequency, err_convert := time.ParseDuration(cmd.Frequency)
```

Comment:
I'm not sure we need to introduce another err variable here. Any reason for introducing `err_convert` and returning `err`?

Reply:
That is definitely a bug. I will fix that now.